requirements.txt

```diff
@@ -2,6 +2,7 @@ anyconfig>=0.9.7, <1.0
 click<8.0
 cookiecutter>=1.6.0, <2.0
 memory_profiler>=0.50.0, <1.0
+networkx<2.4
```

Comment:
This should be moved to `test_requirements.txt` (when users want to use any datasets in `contrib`, they have to pip install the relevant dependency).